devito/core/gpu.py

```diff
@@ -185,6 +188,10 @@ def _specialize_iet(cls, graph, **kwargs):
         # Distributed-memory parallelism
         mpiize(graph, sregistry=sregistry, options=options)
 
+        # Loop tiling
+        if options['blocklevels']:
```

Comment:
does it fail without this?

Reply:
Should be fine to remove yes? Unless we miss sth.

Reply:
You're right. It works fine without the conditional. So I'm removing it for the next commit.